Jenkinsfile.promote

```diff
@@ -92,6 +92,53 @@ pipeline {
             }
         }
 
+        stage('Upload documentation and distribution') {
+            when {
+                expression { return isRelease() }
+            }
+            steps {
+                script {
+                    mavenCleanInstall('optaplanner', true, ['full'])
+                    dir('optaplanner') {
+                        withCredentials(bindings: [sshUserPrivateKey(credentialsId: 'optaplanner-filemgmt',
+                                             keyFileVariable: 'SSH_KEY_JBOSS_FILEMGMT')]) {
+                            // For testing, simulate connection via SSH:
+                            // sh "ssh -i $SSH_KEY_JBOSS_FILEMGMT -oKexAlgorithms=+diffie-hellman-group1-sha1 optaplanner@filemgmt.jboss.org"
+                            sh "./build/release/upload_distribution.sh ${getProjectVersion()} $SSH_KEY_JBOSS_FILEMGMT"
+                        }
+                    }
+                }
+            }
+        }
+
+        stage('Update OptaPlanner website') {
+            when {
+                expression { return isRelease() }
+            }
+            steps {
+                script {
+                    final String websiteRepository = 'optaplanner-website'
+                    dir(websiteRepository) {
+                        String prBranchName = createWebsitePrBranch(websiteRepository)
+
+                        // Update versions in links on the website.
+                        String nextMinorSnapshot = util.getNextVersion(getProjectVersion(), 'minor')
+                        sh "./build/update-versions.sh ${getProjectVersion()} $nextMinorSnapshot"
+
+                        // Update the XSDs. OptaPlanner must be cloned and build with the full profile before.
+                        String optaplannerRoot = "$WORKSPACE/optaplanner"
+                        sh "cp $optaplannerRoot/optaplanner-core/target/classes/solver.xsd xsd/solver/solver-8.xsd"
+                        sh "cp $optaplannerRoot/optaplanner-benchmark/target/classes/benchmark.xsd xsd/benchmark/benchmark-8.xsd"
```

Comment:
`-8.xsd` suffixes are for version 8.x explicitly ?

Reply:
Yep, and there is a symlink solver.xsd pointing to the solver-8.xsd. The point is to have XSDs for the entire 8.x, so that once we head towards 9.x, users staying on 8.x have compatible XSD already available.

Reply:
Maybe we could set this dynamic then, with the version number. Because once we change to version 9, we may forget to update that part (if still there)